apps/test-pid/include/control.h

```diff
@@ -0,0 +1,420 @@
+#pragma once
+#include <array>
+constexpr std::array<float, 5001> control = {
```

Comment:
theres a comment at the top of main.cpp that explains it. i dont think we should put matlab code here

Reply:
i meant commit the matlab code somewhere maybe if we want to adjust the values or get new values. 

also there should be a comment in this file because its where the values are actually stored. comment in main probably only needs to say that its comparing against some ground truth